eatin-client/src/components/functions/SavedStateInRecipes.ts

```diff
@@ -0,0 +1,102 @@
+import { useCallback, useEffect, useState } from "react";
+import { useAuth } from "../../context/auth-context";
+import { useGetUserrecipesByUserIdQuery } from "../../generated/graphql";
+import { Recipe, RecipesSection } from "../types";
+
+export function useAddIsSavedToRecipesSection(givenRecipes: RecipesSection[]) {
```

Comment:
change the file name from `SavedStateInRecipes` to `useAddIsSavedToRecipesSection`